### DISCUSSION TITLE
Tables

### DISCUSSION THREAD

**Discussion by pseudoparenchymatous** [nvim-neorg/neorg#970] [Q&A]
How to translate a table like this to `norg`? [This](https://github.com/nvim-neorg/norg-specs/blob/35a52c4510f3deabfe52112836504ec7f8fa08fd/1.0-specification.norg#L159) and [this](https://github.com/nvim-neorg/norg-specs/blob/35a52c4510f3deabfe52112836504ec7f8fa08fd/1.0-specification.norg#L446) doesn't make sense to me 😭

Edit: I kinda understand those now. If I understand correctly this is how to do it. 
```
: . : color
: _ : hue
: > : light
: > : dark
: _ : red
: > : light-red
: > : dark-red
: _ : green
: > : light-green
: > : dark-green
```

Now I'm interested in learning more about what [this line](https://github.com/nvim-neorg/norg-specs/blob/35a52c4510f3deabfe52112836504ec7f8fa08fd/1.0-semantics.norg#L429) meant or how to do it. 

![IMG_20230701_184921](https://github.com/nvim-neorg/neorg/assets/105337186/1f282678-f483-4b3e-a1c8-2e0b99f2ed31)

**Comment by PlexSheep** [comment]
I am super confused by this too. The specification mentions tables, but I can't get anything related to them to work.

**Reply by PlexSheep** [reply]
I'd be happy if I could just use [table mode](https://github.com/dhruvasagar/vim-table-mode), but sadly that plugin is confused by the markup characters.

**Comment by alexmozaidze** [comment]
I believe you can do
```norg
@table
Write a normal markdown table here...
@end
```

I'm not sure wether it's a good method or not when it comes to doing tables, but it exports to markdown just fine, so I guess it's ok

**Comment by alex-w-k** [comment]
Yeah, I am very confused by tables and how they exist right now, it would be great to have a better way to put tables in and/or improvements to the concealer to how they are displayed after they are typed up

**Comment by pysan3** [comment]
Wrote a tutorial on the table syntax.

https://github.com/pysan3/Norg-Tutorial/blob/main/norg_tutorial.md#table-syntax

I hope this can help you.

Of course this markdown is generated from norg file so check out the source as well.
